packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -417,6 +420,17 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
   /// {@endtemplate}
   final bool transitionBetweenRoutes;
 
+  /// {@template flutter.cupertino.CupertinoNavigationBar.enableBackgroundFilterBlur}
+  /// Whether to have blur effect with non opaque color.
```

Comment:
Nit: This opener feels a little non descriptive to me. Maybe something like:
```suggestion
  /// Whether to have a blur effect when a non-opaque background color is used.
```
might be more descriptive? Feel free to tweak as desired.

Reply:
@victorsanni Thanks for the feedback